client/src/styles/components/dropdown-menu.css

```diff
@@ -13,7 +13,8 @@ Styleguide Components.Dropdown
     border-radius: 0;
     margin-top: 11px;
     padding: 0;
-    width: 200px;
+    min-width: 200px;
+    width: auto;
```

Comment:
Should we have a max-width, for very long folder names?

Reply:
The path folder names won't be an issue bc they're being restricted to 20 chars. So it would only be an issue with system names/labels. My thinking is if we include a max-width the text overflow would happen again with a long system name. I know there's already code in the css file to handle media/device screen size rules. Do we think that's enough to handle the potential issue, or should I add a max-width of 800px and assume we'd never have a super long system name label past that?

Reply:
Ah perfect, I think this is a good solution to this problem. Very long system names are unlikely, but we could address that in another task if need be.

Reply:
Sounds good!